csharp/ql/test/library-tests/dataflow/defuse/defUseEquivalence.ql

```diff
@@ -4,9 +4,9 @@ import csharp
 predicate defReaches(AssignableDefinition def, LocalScopeVariable v, ControlFlow::Node cfn) {
   def.getTarget() = v and cfn = def.getAControlFlowNode().getASuccessor()
   or
-  exists(ControlFlow::Node mid |
-    defReaches(def, v, mid) |
-    not mid = any(AssignableDefinition ad | ad.getTarget() = v and ad.isCertain()).getAControlFlowNode() and
+  exists(ControlFlow::Node mid | defReaches(def, v, mid) |
+    not mid = any(AssignableDefinition ad | ad.getTarget() = v and ad.isCertain())
+          .getAControlFlowNode() and
```

Comment:
@aschackmull : Is the indentation here as expected?

Reply:
Yes. Again this is a case where several expression elements each add indentation levels.  The equality adds one level and the split of the qualifier and the call adds two.  If the long `any(..).getAControlFlowNode()` were on the left-hand side of an equality it would be more clear why this is needed in general to separate things.